crates/contracts/build.rs

```diff
@@ -115,6 +115,12 @@ fn main() {
             "on_swap_with_balances"
         )
     });
+    generate_contract_with_config("BaoswapFactory", |builder| {
+        builder.add_network_str("100", "0x45DE240fbE2077dd3e711299538A09854FAE9c9b")
+    });
+    generate_contract_with_config("BaoswapRouter", |builder| {
+        builder.add_network_str("100", "0x6093AeBAC87d62b1A5a4cEec91204e35020E38bE")
+    });
```

Comment:
The only real changes in this file, the other are just `sort`-ing by contract name.